proto/cosmos/staking/v1beta1/genesis.proto

```diff
@@ -51,3 +117,24 @@ message LastValidatorPower {
   // power defines the power of the validator.
   int64 power = 2;
 }
+
+// Development plan
+// — Implement changes for Msgs flow
+// — — There are 6 queues to be newly added for epoching process and should write a common utility to manage queues by considering existing staking module codebase
+// — — Implement Msgs handling one by one (implement instant change fields, queue actions for epoching)
+// — Implement logic flow after queue action
+// — Implement CLI commands for querying
+// — — BufferedValidators
```

Comment:
Why is this a queue?

Reply:
It's not queue but it's going to Return what the new validator set would be if the epoch ended right now